packages/shared/src/components/profile/SocialChips.tsx

```diff
@@ -44,11 +44,11 @@ export function SocialChips({ links }: SocialChipsProps): ReactElement {
     .filter((key) => !!links[key])
     .map((key) => (
       <Button
-        spanClassName="w-fit my-2 font-normal"
+        spanClassName="my-2 font-normal text-ellipsis whitespace-nowrap overflow-hidden flex-1"
```

Comment:
Dang see this is passed through, else we could use `getTextEllipsis` function/component.

Reply:
Sorry I auto merged. But I see that `getTextEllipsis` returns a component which I can't use as I'm using the button element. With that said, I believe having it by default in the button element can be super beneficial

Reply:
Hmm yeah could be useful as the default on buttons indeed!